test/conformance/chainsaw/validate/clusterpolicy/cornercases/validate-pattern-should-pass/report-pass-assert.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: wgpolicyk8s.io/v1alpha2
+kind: PolicyReport
+metadata:
+  ownerReferences:
+  - apiVersion: v1
+    kind: Pod
+    name: test-pod
+scope:
+  apiVersion: v1
+  kind: Pod
+  name: test-pod
+results:
+- message: validation rule 'priv-esc' passed.
+  policy: priv
+  result: pass
+  rule: priv-esc
+  source: kyverno
+summary:
+  error: 0
+  fail: 0
+  pass: 1
```

Comment:
Do we expect a skip in another failed test scenario?

Reply:
No, in another failed test scenario, we expect just fail

Reply:
I should have made this clearer, I meant to have a third test to check the report with the `skip` result, this means all conditions in the rule are skipped and that leads to the final `skip` status in the report. Does that make sense?

Reply:
Yes, that's how I understood it at first, but then I may have misunderstood the meaning of some of your comment. I'll add this third test later.